Mjml.Net/Components/Body/TableComponent.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Mjml.Net.Components.Body
+{
+    public partial class TableComponent : Component
+    {
+        public override string ComponentName => "mj-table";
+
+        public override ContentType ContentType => ContentType.Raw;
+
+        [Bind("align", BindType.Align)]
+        public string Align = "left";
+
+        [Bind("border", BindType.String)]
+        public string Border = "none";
+
+        [Bind("cellpadding", BindType.String)]
+        public string CellPadding = "0";
+
+        [Bind("cellspacing", BindType.String)]
+        public string CellSpacing = "0";
+
+        [Bind("color", BindType.Color)]
+        public string Color = "#000000";
+
+        [Bind("container-background-color", BindType.Color)]
+        public string? ContainerBackgroundColor;
+
+        [Bind("font-family", BindType.String)]
+        public string FontFamily = "Ubuntu, Helvetica, Arial, sans-serif";
+
+        [Bind("font-size", BindType.Pixels)]
+        public string FontSize = "13px";
+
+        [Bind("font-weight", BindType.String)]
+        public string? FontWeight;
+
+        [Bind("line-height", BindType.PixelsOrPercent)]
+        public string LineHeight = "22px";
+
+        [Bind("padding", BindType.PixelsOrPercent)]
+        public string Padding = "10px 25px";
+
+        [Bind("padding-bottom", BindType.PixelsOrPercent)]
+        public string? PaddingBottom;
+
+        [Bind("padding-left", BindType.PixelsOrPercent)]
+        public string? PaddingLeft;
+
+        [Bind("padding-right", BindType.PixelsOrPercent)]
+        public string? PaddingRight;
+
+        [Bind("padding-top", BindType.PixelsOrPercent)]
+        public string? PaddingTop;
+
+        [Bind("role", BindType.String)]
+        public string? Role;
+
+        [Bind("table-layout", BindType.SocialTableLayout)]
+        public string TableLayout = "auto";
+
+        [Bind("vertical-align", BindType.VerticalAlign)]
+        public string? VerticalAlign;
+
+        [Bind("width", BindType.PixelsOrPercent)]
+        public string Width = "100%";
+
+        public override void Render(IHtmlRenderer renderer, GlobalContext context)
+        {
+
+            var widthValue = 100d;
+            var widthUnit = Unit.Percent;
+            var widthString = "100%";
+
+            if (Width != null)
```

Comment:
Width can never be null, because it has a default value.

Reply:
(facepalm).. thanks ;)